docs/developers/building.rst

```diff
@@ -0,0 +1,133 @@
+======================================
+Build and install MicroOVN from source
+======================================
+
+This how-to contains steps needed for building MicroOVN form its source code.
+This is useful, for example, if you want to contribute to the MicroOVN and you
+want to test your changes locally.
+
+Build requirements
+------------------
+
+MicroOVN is distributed as a snap and the only requirements for building it
+are ``Make`` and ``snapcraft``. You can install them with:
+
+.. code-block:: none
+
+   sudo apt install make
+   sudo snap install snapcraft --classic
+
+Snapcraft requires ``LXD`` to build snaps. So if your system does not have LXD
+installed and initiated, you can check out either `LXD getting started
+guides`_ or go with following default setup:
+
+.. code-block:: none
+
+   sudo snap install lxd
+   lxd init --auto
+
+Build MicroOVN
+--------------
+
+To build MicroOVN, go into the repository's root directory and run:
+
+.. code-block:: none
+
+   make
+
+This will produce the ``microovn.snap`` file that can be then used to install
+MicroOVN on your system.
+
+Install MicroOVN
+----------------
+
+Using the ``microovn.snap`` file created in the previous section, you can
+install MicroOVN in this way:
+
+.. code-block:: none
+
+   sudo snap install --dangerous ./microovn.snap
+
+.. note::
+
+   If you are building latest MicroOVN from the ``main`` branch, it's possible
+   that it's using a non-stable core snap as its base. In that case, you may
+   get a message like this:
+
+   .. code-block:: none
+
+      Ensure prerequisites for "microovn" are available (cannot install snap base "core24": no snap revision available as specified)
+
+   In such a case, you will need to install the required core snap manually
+   from the ``edge`` risk level. For example:
+
+   .. code-block:: none
+
+      snap install core24 --edge
+
+   Then repeat the installation step.
+
+You will also need to manually connect required plugs, as ``snapd`` won't
+do it automatically for locally installed snaps.
+
+.. code-block:: none
+
+   # Switch to root account
```

Comment:
is prefixing the command with `sudo` insufficient?

Reply:
It is sufficient. I'm not sure why I went with this approach